src/release-specification.ts

```diff
@@ -301,6 +300,72 @@ export async function validateReleaseSpecification(
           });
         }
       }
+
+      // Check to compel users to release packages with breaking changes alongside their dependents
+      if (
+        versionSpecifierOrDirective === 'major' ||
+        (isValidSemver(versionSpecifierOrDirective) &&
+          diff(
+            pkg.validatedManifest.version,
```

Comment:
I noticed that the other PR checks that `pkg` is truthy first before using it. Would the same check be appropriate here? We are using it on this line